### PR TITLE
doc: add pipeline guild for supply chain security

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

part of #493 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/content/en/docs/pipeline/chain-security.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+    - name: build-and-push-image
+      predefinedTask:
+        name: build-and-push-image
+        params:
+          image: "ghcr.io/<username>/<image-name>"'| kubectl apply -f -
+```
+
+Replace `<username>` with your GitHub username and `<image-name>` with your image name, like `kurator-test:0.4.1`.
+
+### Exposing Services
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/pipeline/chain-security.md:138]: Should be clear this service if for webhook, but not every one know that

File: docs/content/en/docs/pipeline/chain-security.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+
+Set up a webhook for this pipeline as well, following the detailed process in [Setting Up Your Pipeline](https://kurator.dev/docs/pipeline/setting/).
+
+### Triggering the Pipeline
+
+To trigger the pipeline, consider pushing some content to the repository, such as a modification to the README file. 
+The information about the received event can be observed in the terminal where the port-forwarding service is running:
+
+```console
+Forwarding from 0.0.0.0:30002 -> 8080
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/pipeline/chain-security.md:156]: This is for your kind env, do you mention it before?

File: docs/content/en/docs/pipeline/chain-security.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+### Viewing Pipeline Execution Results
+
+After the pipeline is triggered, individual pods will be created for each task in the pipeline, executing them sequentially. 
+You can view the status of each task's execution with specific commands.
+
+Similarly, after obtaining the pipeline execution name from the command `kurator pipeline execution list -n kurator-pipeline --kubeconfig /root/.kube/kurator-host.config`, 
+you can view the execution logs of each task in the pipeline using the following command:
+
+```console
+$ kurator pipeline execution logs <pipeline-execution>  -n kurator-pipeline --tail 10 --kubeconfig /root/.kube/kurator-host.config
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/pipeline/chain-security.md:169]: can you provide a command to get this name

File: docs/content/en/docs/pipeline/chain-security.md [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+cosign verify-attestation --key cosign.pub --type slsaprovenance ghcr.io/<username>/<repository-name>
+```
+
+If verification fails, explicit error messages will be displayed (e.g., signature mismatch, invalid forepart). 
+If successful, detailed information about the signature, including the docker image, will be shown.
+
+Here's an example of a successful verification output:
+
+```console
+$ cosign verify --key cosign.pub ghcr.io/<username>/<repository-name>
```
[the remark this answers is not in the document]
Comment (Xieql) [reply docs/content/en/docs/pipeline/chain-security.md:267]: it is about image, but  `<repository-name>` is different to  `<image-name>`, `kurator` is a repository name and the  `kurator:0.6.0` is image name

[the remark this answers is not in the document]
Comment (Xieql) [reply docs/content/en/docs/pipeline/chain-security.md:267]: The differences are strictly distinguished in my document.

File: docs/content/en/docs/pipeline/chain-security.md (hunk 3 above)
Comment (Xieql) [reply docs/content/en/docs/pipeline/chain-security.md:169]: `kurator pipeline execution list -n kurator-pipeline --kubeconfig /root/.kube/kurator-host.config`, 
I've already demonstrated this command earlier, and its usage has appeared in previous documents.
I can emphasize this command using console notation.

File: docs/content/en/docs/pipeline/chain-security.md (hunk 2 above)
Comment (Xieql) [reply docs/content/en/docs/pipeline/chain-security.md:156]: Yes，I need add those link

File: docs/content/en/docs/pipeline/chain-security.md (hunk 1 above)
Comment (Xieql) [reply docs/content/en/docs/pipeline/chain-security.md:138]: in last  doc, I already explain it, may be we can add link for it.

File: docs/content/en/docs/pipeline/chain-security.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (hzxuzhonghu) [reply docs/content/en/docs/pipeline/chain-security.md:272]: donot we need a tag?

[the remark this answers is not in the document]
Comment (hzxuzhonghu) [reply docs/content/en/docs/pipeline/chain-security.md:272]: I think we can use a <Image> here, not user/repo.

[the remark this answers is not in the document]
Comment (Xieql) [reply docs/content/en/docs/pipeline/chain-security.md]: ok

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**